cmd/pow/cmd/deals_store.go

```diff
@@ -8,13 +8,15 @@ import (
 	"strconv"
```

Comment:
I'm not sure the low-level Store API would be useful for people. Expecting someone to provide the size of the stored thing to calculate the right price sounds very demanding to do "manually".